docs/docs/started/cloud-providers.md

```diff
@@ -0,0 +1,61 @@
+---
+id: cloud-providers
+---
+
+# Cloud providers
+
+A selection of Nebari supported cloud providers
+
+Now that Nebari is successfully installed we can move forward with choosing the best suitable cloud provider to help you effectively deploy and start your projects within Nebari.
+
+To deploy Nebari, each cloud provider requires fairly wide permissions to create all the necessary cloud resources provided in the form of `access keys`. Hence, once the Cloud provider has been chosen, follow the steps described in their respectful How-to guides to move forward to the deployment.
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+  <TabItem value="gcp" label="Google GCP" default>
+
+<div class="text--center">
+  <img src="/img/started-google-cloud-logo.png" width={420} />
+</div>
+
+[Google GCP](https://docs.qhub.dev/en/latest/source/installation/setup.html#google-cloud-platform) (also known as Google Cloud Platform or GCP) is a provider of computing resources for developing, deploying, and operating computing services.
```

Comment:
I think it would be good to rethink how the attributes of the various cloud providers are presented here. As they are now, they're kind of scattered:

- GCP is introduced only very vaguely
- AWS gets a longer description and mention of IaaS, PaaS, etc.
- Digital Ocean gets mention of 'cloud computing' (seems like too general a term?) and IaaS, and mention of 'scalability and pricing' (do we want to make these sorts of statements, that might imply that other cloud providers are pricey or poorly scalable?)

And so on.  

I think a more clinical presentation of the features present/absent in each would be better, especially having the features for each laid out in the same order, and with the same terminology/acronyms/etc.

Reply:
They are all one sentence explanations. Can you open an issue to address this in a separate PRR?